widget_driver/example/lib/widgets/playground/playground_test_widget/playground_test_widget.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:widget_driver/widget_driver.dart';
+
+import 'playground_test_child_widget.dart';
+import 'playground_test_widget_driver.dart';
+
+class PlaygroundTestWidget extends DrivableWidget<PlaygroundTestWidgetDriver> {
```

Comment:
This new widget also only exists to show case have to use the new TestConfigProvider. So that you can tell it to use real driver for all driver. And also only use real driver for its child widget called `PlaygroundTestChildWidget`